convert-stage-4.sh

```diff
@@ -130,6 +132,9 @@ install_files() {
   sudo install -d -m 755 ${data_dir}/${dataconfdir}
   sudo install -d -m 755 ${data_dir}/${databootdir}
 
+  install -d -m 755 ${primary_dir}/${sysconfdir}/scripts/
+  echo -n "${mender_state_scripts_version}" > ${primary_dir}/${sysconfdir}/scripts/version
```

Comment:
Okey, so the error in Jenkins **is** related to this change.

When running the script outside from Docker, you need sudo to write here. So you need to replace this with something like:
```
echo -n "${mender_state_scripts_version}" | sudo tee ${primary_dir}/${sysconfdir}/scripts/version
```

Reply:
Indeed, good tests :)